crates/modelardb_compression/src/lib.rs

```diff
@@ -133,9 +133,12 @@ pub fn try_compress(
     Ok(compressed_record_batch_builder.finish())
 }
 
-/// Create a single compressed segment that store `maybe_selected_model` and any residuals; two
-/// compressed segments, with the first storing `maybe_selected_model` and the second storing
-/// residuals; or a compressed segment that store residuals if `maybe_selected_model` is [`None`].
+/// Create segment(s) that store `maybe_selected_model` and residuals as either:
+/// - One compressed segment that stores `maybe_selected_model` and a small number of residuals.
+/// - Two compressed segments with the first storing `maybe_selected_model` and the second storing
+/// a large number of residuals. 
```

Comment:
Maybe change this so it mentions specifically how many residuals there can be instead of `a small number` and `a large number`.

Reply:
Fixed.